internal/controller/rsct_controller.go

```diff
@@ -71,10 +71,15 @@ func (r *RSCTReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		return reconcile.Result{}, fmt.Errorf("failed to get RSCT %s: %w", req, err)
 	}
 
-	// TODO(mjturek): Allow image specification
 	r.Config.Namespace = rsct.Namespace
 	r.Config.Name = rsct.Name
-	r.Config.Image = "quay.io/powercloud/rsct-ppc64le:latest"
+
+	// Using the default RSCT image if not specified in RSCTSpec
+	if *rsct.Spec.Image == "" {
```

Comment:
check for nil check before checking for empty string

Reply:
Added check for nil instead of empty string

Reply:
Now, it is fetching default value. If the spec is empty, the default value will be used.
```
apiVersion: rsct.ibm.com/v1alpha1
kind: RSCT
metadata:
  name: rsct
  namespace: rsct-operator-system
spec: {}
```